recipe/meta.yaml

```diff
@@ -68,21 +68,20 @@ outputs:
         - python
         - aiosignal
         - attrs
-        - click >=7.0, <=8.0.4
+        - click >=7.0
         - colorama
         - filelock
         - frozenlist
-        # We override the 1.43.0 upper-bound in setup.py here,
-        # see https://github.com/ray-project/ray/issues/28354
-        - grpcio >=1.32.0,<1.48
+        # mattip/h-vetinari [2022/11]: unclear problems with conda-forge's 1.49
+        - grpcio <1.49.0
```

Comment:
I wonder why we don't impose any lower bound here, maybe we should set it to `>=1.42,<1.49` (taking lower bound from `setup.py` for py3.10)?

Reply:
Yeah, we can use the lower bound. Not that it matters much, I doubt the solver would ever realistically reach that low.

Reply:
OK